admin/admin-filters-term.php

```diff
@@ -658,37 +658,39 @@ public function split_shared_term( $term_id, $new_term_id, $term_taxonomy_id, $t
 	public function get_subsequently_inserted_term_language( $slug, $taxonomy ) {
 		$lang = false;
 
-		if ( isset( $_POST['term_lang_choice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			$lang = $this->model->get_language( sanitize_key( $_POST['term_lang_choice'] ) ); // phpcs:ignore WordPress.Security.NonceVerification
+		if ( ! empty( $_POST['term_lang_choice'] ) && is_string( $_POST['term_lang_choice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			$lang_slug = sanitize_key( $_POST['term_lang_choice'] ); // phpcs:ignore WordPress.Security.NonceVerification
+			if ( ! empty( $lang_slug ) ) {
+				$lang = $this->model->get_language( $lang_slug );
+			}
 		}
 
-		elseif ( isset( $_POST['inline_lang_choice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			$lang = $this->model->get_language( sanitize_key( $_POST['inline_lang_choice'] ) ); // phpcs:ignore WordPress.Security.NonceVerification
+		elseif ( ! empty( $_POST['inline_lang_choice'] ) && is_string( $_POST['inline_lang_choice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			$lang_slug = sanitize_key( $_POST['inline_lang_choice'] ); // phpcs:ignore WordPress.Security.NonceVerification
+			if ( ! empty( $lang_slug ) ) {
+				$lang = $this->model->get_language( $lang_slug );
+			}
 		}
 
 		// *Post* bulk edit, in case a new term is created
 		elseif ( isset( $_GET['bulk_edit'], $_GET['inline_lang_choice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
 			// Bulk edit does not modify the language
-			if ( -1 == $_GET['inline_lang_choice'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			if ( -1 === (int) $_GET['inline_lang_choice'] ) { // phpcs:ignore WordPress.Security.NonceVerification
 				$lang = $this->model->post->get_language( $this->post_id );
 			} else {
```

Comment:
To be consistent:
```suggestion
			} elseif ( is_string( $_POST['inline_lang_choice'] ) ) {
```

Maybe I worry too much but I don't trust `$_GET` and `$_POST` 😅

Reply:
Man of little faith! 😝